lib/salus/scanners/osv/base.rb

```diff
@@ -0,0 +1,152 @@
+require 'salus/scanners/base'
+require 'json'
+require 'zip'
+require 'uri'
+require 'net/http'
+
+module Salus::Scanners::OSV
+  class Base < Salus::Scanners::Base
+    def run
+      raise NoMethodError, 'implement in subclass'
+    end
+
+    def should_run?
+      raise NoMethodError, 'implement in subclass'
+    end
+
+    def name
+      self.class.name.sub('Salus::Scanners::OSV::', '')
+    end
+
+    private
+
+    def osv_vulnerabilities
+      @osv_vulnerabilities ||= fetch_vulnerabilities
+    end
+
+    def osv_url
```

Comment:
def osv_url -> def osv_urls

Because, the method is returning an array of urls

Reply:
Good point